orca-retrofit/src/main/groovy/com/netflix/spinnaker/orca/retrofit/exceptions/RetrofitExceptionHandler.groovy

```diff
@@ -56,8 +60,31 @@ class RetrofitExceptionHandler implements ExceptionHandler<RetrofitError> {
       response.details.kind = e.kind
       response.details.status = properties.status ?: null
       response.details.url = properties.url ?: null
-      response.shouldRetry = (e.kind == RetrofitError.Kind.NETWORK)
+      response.shouldRetry = (e.kind == NETWORK && findHttpMethodAnnotation(e) in ["GET", "HEAD"])
```

Comment:
Yeah, I was wondering about DELETE specifically. I think the problem with PUT is it gets used wrong as often as it gets used right. I know I mix up PUT and POST all the time.